sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -1188,13 +1188,18 @@ def test_set_expiry(self, **kwargs):
             content_disposition='inline')
         expiry_time = self.get_datetime_variable(variables, 'expiry_time', datetime.utcnow() + timedelta(hours=1))
         file_client = directory_client.create_file("newfile", metadata=metadata, content_settings=content_settings)
+
+        # Act / Assert
         file_client.set_file_expiry("Absolute", expires_on=expiry_time)
         properties = file_client.get_file_properties()
-
-        # Assert
         assert properties
         assert properties.expiry_time is not None
 
+        file_client.set_file_expiry("NeverExpire")
+        properties = file_client.get_file_properties()
+        assert properties
+        assert properties.expiry_time is None
```

Comment:
qq: Do you think it would be worthwhile to assert that this `properties.expiry_time` is not equal to the one above, which was the result of `file_client.set_file_expiry("Absolute", expires_on=expiry_time)`?

Only reason being, I think technically this test could pass even if `file_client.set_file_expiry("NeverExpire")` took no effect, since it was already made non-None above by setting it to `Absolute`.

Reply:
Follow-up: If we `assert never_expire_properties.expiry_time != absolute_properties.expiry_time`, wouldn't this be equivalent to the existing code since `never_expire_properties.expiry_time is None` and `absolute_properties.expiry_time is not None`?

Reply:
This looks fine as-is.